packages/webcert/src/store/certificate/certificateMiddleware.test.ts

```diff
@@ -477,6 +481,26 @@ describe('Test certificate middleware', () => {
     })
   })
 
+  describe('Handle create certificate', () => {
+    it('should update certificate id after api call', async () => {
+      const data: CreateCertificate = {
+        certificateType: 'lisjp',
+        patientId: '191212121212',
+      }
+      const response: CreateCertificateResponse = {
+        certificateId: 'certificateId',
+      }
+
+      fakeAxios.onPost(`/api/certificate/${data.certificateType}/${data.patientId}`).reply(200, response)
+
+      testStore.dispatch(createNewCertificate(data))
+
+      await flushPromises()
+      expect(testStore.getState().ui.uiCertificate.createdCertificateId).toEqual(response.certificateId)
+      expect(fakeAxios.history.post.length).toBe(1)
+    })
+  })
```

Comment:
Should be separate test for success

Reply:
Fixed, but I'm not sure if I did it correctly. Please check.